.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### What is the context of this PR?
+Describe what you have changed and why, link to other PRs or Issues as appropriate.
+
+### How to review 
+Describe the steps required to test the changes (include screenshots if appropriate).
+
+### Checklist
+
+* [ ] New static content marked up for translation
+* [ ] Jsonnet files conform to the latest style guide
```

Comment:
Do we need this?. Even if we decide yes, the first bullet should be removed.

Reply:
So I removed first bullet, second one has a link now, it's not my idea to have this checklist here, it's on the [original trello card](https://trello.com/c/jbSBLVgz).